Find a functionally equivalent code:android.service.dreams.DreamService.setLowProfile:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Sets View.SYSTEM_UI_FLAG_LOW_PROFILE on the content view.
 *
 * @param lowProfile True to set View.SYSTEM_UI_FLAG_LOW_PROFILE
 * @hide There is no reason to have this -- dreams can set this flag
 * on their own content view, and from there can actually do the
 * correct interactions with it (seeing when it is cleared etc).
 */

@@ -1,5 +1,7 @@
 {
-    mLowProfile = lowProfile;
-    int flag = View.SYSTEM_UI_FLAG_LOW_PROFILE;
-    applySystemUiVisibilityFlags(mLowProfile ? flag : 0, flag);
+    if (mLowProfile != lowProfile) {
+        mLowProfile = lowProfile;
+        int flag = View.SYSTEM_UI_FLAG_LOW_PROFILE;
+        applySystemUiVisibilityFlags(mLowProfile ? flag : 0, flag);
+    }
 }

